api_testing/test_getting_resource.py

```diff
@@ -0,0 +1,12 @@
+from schemas.data_preparation import ValidationJsonSchemas
+
+
+def test_get_user_1(session, base_url):
+    get_user_1 = session.get(base_url + '1')
+    assert get_user_1.status_code == 200
+    ValidationJsonSchemas.checking_json_schema(url=get_user_1)
```

Comment:
You can add test case for todo with max id (200) to complete boundary value analysis. Also you could add check API response for correct todo id:

```
@pytest.mark.parametrize('todo_id', [1, 200])
def test_get_todo(session, base_url, todo_id):
    res = session.get(base_url + str(todo_id))
    assert res.status_code == 200
    assert res.json()['id'] == todo_id
    ValidationJsonSchemas.checking_json_schema(url=res)
```